kopi/chain.json

```diff
@@ -0,0 +1,287 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "kopi",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://app.kopi.money",
+  "pretty_name": "Kopi",
+  "chain_type": "cosmos",
+  "chain_id": "luwak-1",
+  "daemon_name": "kopid",
+  "node_home": "$HOME/.kopid",
+  "bech32_prefix": "kopi",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ukopi",
+        "low_gas_price": 0.01,
+        "average_gas_price":  0.025,
+        "high_gas_price": 0.03
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ukopi"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/kopi-money/kopi",
+    "recommended_version": "v0.6.5.1",
+    "genesis": {
+      "genesis_url": "https://data.kopi.money/genesis.json"
+    },
+    "versions": [
```

Comment:
`versions` isn't recorded in `codebase` anymore, it's now a separate json file `"versions.json"`